OpenXMLSDK.Engine/Word/ReportEngine/Models/Cell.cs

```diff
@@ -19,6 +19,11 @@ public class Cell : BaseElement
         /// </summary>
         public int ColSpan { get; set; }
 
+        /// <summary>
+        /// DoubleModelobject linked that will be linked to the ColSpan property
```

Comment:
Can you update the summary and use <see in order to create a direct link to DoubleModel class ?